### PR TITLE
Rn aseq modules

### PR DESCRIPTION
**Pull request by tilschaef**
- RNASeq procceses

### CODE REVIEW

File: HTSeq/0.6.0/htseqcount.nf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+process htseqcount {
+    tag {"htseq-count ${sample_id}"}
+    label 'HTSeq_0_6_0'
+    label 'htseq_count_0_6_0'
```
Comment (rernst) [inline HTSeq/0.6.0/htseqcount.nf]: Should be: 'HTSeq_0_6_0_count'

File: Sambamba/0.6.8/Index.nf [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  input:
+    tuple sample_id, file(bams)
+
+  output:
+    tuple sample_id, file("${bams}.bai")
+
+  script:
+  """
+  sambamba index -t ${task.cpus} $bams ${bams}.bai
+  """
```
Comment (rernst) [inline Sambamba/0.6.8/Index.nf]: Does this work for a bam or multiple bams?

File: Sambamba/0.6.8/Index.nf [hunk 3]
```diff
@@ -0,0 +1,20 @@
+process Index {
+  tag {"SAMBAMBA_index ${sample_id}"}
+  label 'SAMBAMBA_index_0_6_8'
```
Comment (rernst) [inline Sambamba/0.6.8/Index.nf]: Label order and missing sambamba version label.

File: Preseq/2.0.3/Preseq.nf [hunk 4]
```diff
@@ -0,0 +1,19 @@
+ process Preseq {
```
Comment (rernst) [inline Preseq/2.0.3/Preseq.nf]: I think this should be called 'lc_extrap'

**Comment by rernst** [comment]
@sawibo 
> Please move container to a separate config
I thought we agreed to put containers in the module files? Otherwise we still have to figure containers out each time we use a module. We should put them in a (hpc) directory were everyone from at least cog_bio can use them. Or use a biocontainer as long as there is no hosted guix container alternative.

File: Sambamba/0.6.8/Index.nf (hunk 2 above)
Comment (tilschaef) [reply Sambamba/0.6.8/Index.nf]: This is indeed a single bam/bai per tuple

**Pull request merged**